### ISSUE TITLE
Rendor can't handle 2 function calls in a function call

### ISSUE THREAD

**Issue by StandingPadAnimations** [StandingPadAnimations/Rendor#12]
In a program like:
```
edef main() 
{
    xyz(input("Enter Args: "), input("Enter Args: "))
}

edef xyz(foo, bar)
{
    echo(foo)
    echo(bar)
}
```
Rendor will end up compiling code like:
```
LOAD 0
DEFINE main
CALL xyz
CALL input
CONST _SEnter Args: 
FINALIZE_CALL input
FINALIZE_CALL xyz
CALL input
CONST _SEnter Args: 
FINALIZE_CALL input
FUNCTION END
DEFINE xyz
ARGUMENT bar
ARGUMENT foo
CALL echo
CONST _&foo
FINALIZE_CALL echo
CALL echo
CONST _&bar
FINALIZE_CALL echo
FUNCTION END
END 0```
Where the second call goes outside the main function call

**Comment by StandingPadAnimations** [comment]
This specific bug has been fixed, but another persists

**Comment by StandingPadAnimations** [comment]
This has been fixed in 7d0fa31b68ae6c3c3b62d15b3cac5a6e7c5eb57f
